datasetcard.md

```diff
@@ -33,9 +33,13 @@ task_ids:
 - {subtask_0}  # Example: extractive-qa
 - {subtask_1}  # Example: multi-class-image-classification
 paperswithcode_id: {paperswithcode_id}  # Dataset id on PapersWithCode (from the URL). Example for SQuAD: squad
-configs:  # Optional for datasets with multiple configurations like glue.
-- {config_0}  # Example for glue: sst2
-- {config_1}  # Example for glue: cola
+configs:  # Optional for datasets with multiple configurations.
+  - config_name: default
+    data_files:
+      - split: train
+        path: "data.csv"
+      - split: test
+        path: "holdout.csv"
```

Comment:
```suggestion
configs:  # Optional. This can be used to pass additional parameters to the dataset loader, such as `data_files`, `data_dir`, and any builder-specific parameters  
  - config_name: {config_name_0}  # Example: default
    data_files:
      - split: {split_name_0}  # Example: train
        path: {file_path_0}  # Example: data.csv
      - split: {split_name_1}  # Example: test
        path: {file_path_1}   # Example: holdout.csv
   - config_name: {config_name_1}  # Example: ...
      ... #TODO
```
and then we can also add example, ideally for some real dataset with multiple configs, maybe copy a part from [nyu-mll/glue](https://huggingface.co/datasets/nyu-mll/glue/blob/main/README.md?code=true#L346)?

Reply:
Looks clearer, thanks. 

About the example: How about using [ELI5 dataset card](https://huggingface.co/datasets/eli5/blob/main/README.md)? I come up because it is already used in [Dataset card creatino guide](https://huggingface.co/docs/hub/datasets-cards#dataset-card-creation-guide). I find it always easier when the same example is referenced. WDYT?